src-ui/src/app/components/manage/settings/settings.component.ts

```diff
@@ -45,17 +51,49 @@ export class SettingsComponent implements OnInit {
   ngOnInit() {
     this.savedViewService.listAll().subscribe(r => {
       this.savedViews = r.results
+      let storeData = {
+        'bulkEditConfirmationDialogs': this.settings.get(SETTINGS_KEYS.BULK_EDIT_CONFIRMATION_DIALOGS),
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jonaswinkler/paperless-ng/pullRequest?prid=7592507)
```suggestion
        "bulkEditConfirmationDialogs": this.settings.get(SETTINGS_KEYS.BULK_EDIT_CONFIRMATION_DIALOGS),
```